omod/src/main/java/org/openmrs/module/datafilter/web/UserFormSubmissionHandler.java

```diff
@@ -41,35 +41,35 @@
 
 @Component("userFormSubmissionHandler")
 public class UserFormSubmissionHandler {
-	
```

Comment:
Why do you have these format changes? Please install the openmrs code formatter in your IDE and use that to format your code, the formatter is in the tools sub module of openmrs core

Reply:
I have installed the Openmrs formatter that has been mentioned last time from openmrs core in IDE. I did format using that and see the above changes. 
Please let me know if I am missing something

![image](https://user-images.githubusercontent.com/40290306/79218469-1a3e1500-7e6e-11ea-9a69-c218dfb3e90c.png)